parser-typechecker/src/Unison/TypePrinter.hs

```diff
@@ -91,12 +91,12 @@ prettyRaw n im p tp = go n im p tp
     Effect1' e t ->
       PP.parenthesizeIf (p >= 10) $ go n im 9 e <> " " <> go n im 10 t
     Effects' es         -> effects (Just es)
-    ForallNamed' v body -> if (p < 0)
+    ForallsNamed' vs body -> if (p < 0 && all Var.isLowercase vs)
```

Comment:
@atacratic just calling your attention to this case, using `ForallsNamed` pattern to pull out nested foralls (which also covers the case of just one variable). The existing code would have inserted multiple forall symbols if they were nested.